feconf.py

```diff
@@ -459,6 +460,10 @@ def get_empty_ratings():
 SUGGESTION_URL_PREFIX = '/suggestionhandler'
 UPLOAD_EXPLORATION_URL = '/contributehandler/upload'
 USERNAME_CHECK_DATA_URL = '/usernamehandler/data'
+FLAG_EXPLORATION_EMAIL_HANDLER_URL = (
```

Comment:
I think I should make the changes after #2437 is merged with develop. Shouldn't I?